frontend/src/pages/AccountSend.vue

```diff
@@ -80,12 +80,19 @@ function useSendForm() {
     return 'Please enter an ENS or CNS name';
   }
 
+  const isEth = (address: string) => getAddress(address) === '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
+
   function isValidTokenAmount(val: string) {
     if (!val || !(Number(val) > 0)) return 'Please enter an amount';
     if (!token.value) return 'Please select a token';
+
     const { address: tokenAddress, decimals } = token.value;
+    if (Number(val) < 0.01 && isEth(tokenAddress)) return 'Please send at least 0.1 ETH';
```

Comment:
Looks like the error message is off. Should read 0.01 ETH, no?

Reply:
Just fixed!